server/src/main/java/org/elasticsearch/client/internal/ClusterAdminClient.java

```diff
@@ -242,6 +245,14 @@ public void nodesCapabilities(final NodesCapabilitiesRequest request, final Acti
         execute(TransportNodesCapabilitiesAction.TYPE, request, listener);
     }
 
+    public ActionFuture<NodesFeaturesResponse> nodesFeatures(final NodesFeaturesRequest request) {
+        return execute(TransportNodesFeaturesAction.TYPE, request);
+    }
+
+    public void nodesFeatures(final NodesFeaturesRequest request, final ActionListener<NodesFeaturesResponse> listener) {
+        execute(TransportNodesFeaturesAction.TYPE, request, listener);
+    }
+
```

Comment:
I would rather not add to this API, especially for an action which is so very specialized. Let's just invoke `Client#execute` directly as needed.